qcodes/plots/qcmatplotlib.py

```diff
@@ -41,6 +43,33 @@ def align_y_axis(ax, ax_target):
     posn_old, posn_target = ax.get_position(), ax_target.get_position()
     ax.set_position([posn_old.x0, posn_target.y0, posn_old.width, posn_target.height])
 
+def set_zscale(self, scale : str):
```

Comment:
Doicstring would be useful here, though I don't always adhere to this as well :-P

Reply:
Agreed.